packages/components/pill/src/styles.ts

```diff
@@ -1,38 +1,23 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
 import { PillVariants } from './types';
-import type { CSSObject } from '@emotion/serialize';
-
-const getCloseButtonStyle = (variant: PillVariants) => {
-  const variantStyle = (variant: PillVariants): CSSObject => {
-    switch (variant) {
-      case 'active':
-        return {
-          borderLeft: `1px solid ${tokens.gray400}`,
-          '&:hover, &:focus': {
-            background: tokens.gray300,
-            cursor: 'pointer',
-          },
-        };
-      default:
-        return {
-          borderLeft: `1px solid ${tokens.gray400}`,
-          '&:hover, &:focus': {
-            background: tokens.gray300,
-            cursor: 'pointer',
-          },
-        };
-    }
-  };
 
+const getCloseButtonStyle = () => {
   return css({
-    border: 'none',
-    borderTopRightRadius: tokens.borderRadiusSmall,
-    borderBottomRightRadius: tokens.borderRadiusSmall,
+    borderLeft: `1px solid ${tokens.gray400}`,
+    borderTopLeftRadius: 0,
+    borderBottomLeftRadius: 0,
     padding: tokens.spacingXs,
-    background: 'transparent',
-    transition: `background ${tokens.transitionDurationShort} ${tokens.transitionEasingDefault}`,
-    ...variantStyle(variant),
+    minHeight: 'auto',
+    transition: `background ${tokens.transitionDurationShort} ${tokens.transitionEasingDefault},
```

Comment:
Probably don't need it, since we have this transition style in `Button`

Reply:
we actually need this, because the button has also `border` transition and we don't need it here.